app/views/groups/_groups_manager.js.jsx.erb

```diff
@@ -280,7 +280,11 @@
       var groups_data = this.props.groups.map(function(group) {
         var g = {};
         g['id'] = group.id;
+        <% if @assignment.scanned_exam %>
+        g['name'] = <a href={'groups/assign_scans?group=' + group.id}>{group.name}</a>;
```

Comment:
@DecathectZero sorry I missed this one. You can use the `Routes` Javascript object to generate routes. Can't remember if you've done that before - https://github.com/railsware/js-routes#usage for examples.